driver-portal/src/Program.cs

```diff
@@ -0,0 +1,74 @@
+
+using System.Text.Json.Serialization;
+using Microsoft.OpenApi.Models;
+
+var builder = WebApplication.CreateBuilder(args);
+
```

Comment:
should be 

builder.WebHost
    .UseUrls()
    .UseKestrel(options =>
    {
    options.Listen(IPAddress.Any, 8080);
    });